spec/feature/authenticated_user_spec.rb

```diff
@@ -130,7 +130,43 @@
     expect(current_path).to eq(new_reservation_path)
   end
 
-  xit "can remove a listing from a cart" do
+  it "cannot see listing intheir cart after it has been requested", js: true do
+    allow_any_instance_of(ApplicationController).to receive(:current_user).
+                                                    and_return(valid_user)
+
+    visit user_listing_path(valid_user.slug, listing)
+    page.execute_script("$('#check_in').val('02/24/2015')")
+    page.execute_script("$('#check_out').val('02/26/2015')")
+    click_button('Book It!')
+    expect(current_path).to eq(user_listing_path(valid_user.slug, listing))
+    expect(page).to have_content('Listing Successfully Added To Reservations')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.